tests/integration/api/controllers/db-doc.spec.js

```diff
@@ -786,7 +786,6 @@ describe('db-doc handler', () => {
               if (reportScenarios[idx].allowed) {
                 chai.expect(result).to.deep.include(reportScenarios[idx].doc);
               } else {
-                console.log(idx, result);
```

Comment:
Request objects are huge and logging them here accounted for most of the lines in the log file...

Reply:
This was probably accidentally committed. thanks for the cleanup!!